client/extensions/zoninator/components/settings/zones-dashboard/index.jsx

```diff
@@ -4,50 +4,44 @@
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import { find, flowRight, get, times } from 'lodash';
+import { flowRight, times } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import HeaderCake from 'components/header-cake';
 import SectionHeader from 'components/section-header';
-import sectionsModule from 'sections';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
 import QueryZones from '../../data/query-zones';
 import ZoneItem from './zone-item';
 import ZonePlaceholder from './zone-placeholder';
 import { getZones, isRequestingZones } from '../../../state/zones/selectors';
+import { settingsPath } from '../../../app/util';
 
 const placeholderCount = 5;
 
-const ZonesDashboard = ( { isRequesting, siteId, siteSlug, translate, zones } ) => {
-	const sections = sectionsModule.get();
-	const section = find( sections, ( value => value.name === 'zoninator' ) );
-	const settingsPath = get( section, 'settings_path' );
+const ZonesDashboard = ( { isRequesting, siteId, siteSlug, translate, zones } ) => (
+	<div>
+		<QueryZones siteId={ siteId } />
 
-	return (
-		<div>
-			<QueryZones siteId={ siteId } />
+		<HeaderCake backHref={ `/plugins/zoninator/${ siteSlug }` }>
+			Zoninator Settings
+		</HeaderCake>
 
-			<HeaderCake backHref={ `/plugins/zoninator/${ siteSlug }` }>
-				Zoninator Settings
-			</HeaderCake>
-
-			<SectionHeader label={ translate( 'Zones' ) }>
-				<Button compact href={ `${ settingsPath }/new/${ siteSlug }` }>
-					{ translate( 'Add a zone' ) }
-				</Button>
-			</SectionHeader>
-			{ isRequesting && zones.length === 0 && times( placeholderCount, i => (
-				<ZonePlaceholder key={ i } />
-			) ) }
-			{ zones.map( ( zone ) => (
-				<ZoneItem key={ zone.slug } zone={ zone } />
-			) ) }
-		</div>
-	);
-};
+		<SectionHeader label={ translate( 'Zones' ) }>
+			<Button compact href={ `${ settingsPath }/new/${ siteSlug }` }>
+				{ translate( 'Add a zone' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **14** times:
`translate( 'Add a DNS Zone' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/zoninator-extension-path-util)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).